zilencer/views.py

```diff
@@ -132,6 +136,31 @@ def register_remote_server(
     except ValidationError as e:
         raise JsonableError(e.message)
 
+    # We don't want to allow disposable domains for contact_email either
+    try:
+        validate_disposable(contact_email)
+    except DisposableEmailError:
+        raise JsonableError(_("Contact email in disposable domains is not allowed"))
+
+    contact_email_domain = Address(addr_spec=contact_email).domain.lower()
+    if contact_email_domain == "example.com":
+        raise JsonableError(_("example.com is not a valid domain for contact email"))
+
+    # Check if the domain has an MX record
+    try:
+        records = DNS.mxlookup(contact_email_domain)
+        dns_ms_check_successful = True
+        if not records:
+            dns_ms_check_successful = False
+    except DNS.Base.ServerError:
+        dns_ms_check_successful = False
+    if not dns_ms_check_successful:
+        raise JsonableError(
+            _("{contact_email_domain} does not have a valid MX record").format(
```

Comment:
Maybe `{contact_email_domain} does not exist or is not configured to accept email`, to be mildly less technical?

Reply:
Let's call the variable `domain` -- may make it easier to reuse the translated string later.